### PR TITLE
Bh-970 large screen layout 

### PR DESCRIPTION
**Pull request by sjv01180**
Here is the layout of the main content for screens greater than 1700 pixels:
image

### CODE REVIEW

File: sass/main.scss [hunk 1]
```diff
@@ -940,6 +938,57 @@ footer {
         grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
+
+}
+
+@media screen and (min-width: 1700px) {
```
Comment (peterkos) [inline sass/main.scss:944]: Please fix all indentation: 4 space throughout!

File: sass/main.scss [hunk 2]
```diff
@@ -940,6 +938,57 @@ footer {
...
+    nav {
+        font-size: 1.5em;
+
+        #login {
+	    padding: 15px 30px;
+	}
+
+	#logo {
```
Comment (peterkos) [inline sass/main.scss:952]: Hmm, this won't change anything except make the logo not aligned with the rest of the content:
Screen Shot 2020-12-20 at 1 37 58 AM

Might be better to leave logo (and navbar, as you've already done) at the same size, looks fine imo

File: sass/main.scss [hunk 3]
```diff
@@ -940,6 +938,57 @@ footer {
...
+	    width: 120px;
+	    height: 120px;
+	}
+
+	#mlh-badge{
```
Comment (peterkos) [inline sass/main.scss:957]: nit
```suggestion
	#mlh-badge {
```

Comment (peterkos) [reply sass/main.scss:957]: The mlh badge being larger, though, looks great!

File: sass/main.scss (hunk 1 above)
Comment (peterkos) [reply sass/main.scss:944]: Also could you move this entire `@media` query to be the first media query? That way it goes largest screen -> smallest screen target.

File: sass/main.scss [hunk 4]
```diff
@@ -940,6 +938,57 @@ footer {
...
+	    width: 140px;
+	}
+
+    }
+
+    #hero {
+        #desk {
```
Comment (peterkos) [inline sass/main.scss:964]: The SVG changes don't seem to have an effect anymore, probably due to our numerous refactors of how scaling works... imo it looks like a good size, the text tweak on `#bh7` is much better for the page "weight". Maybe just remove these SVG adjustments?

File: sass/main.scss [hunk 5]
```diff
@@ -940,6 +938,57 @@ footer {
...
+        #bh7 {
+            font-size: 6.4em;
+        }
+    
+        #premiere {
+            font-size: 3em;
+        }
+
+        #dates {
+            font-size: 1.57em;
```
Comment (peterkos) [inline sass/main.scss:985]: Why _exactly_ `1.57`? `1.6` should be good too -- let's try to stick to tenths place nums unless absolutely necessary.

File: sass/main.scss (hunk 4 above)
Comment (peterkos) [reply sass/main.scss:964]: Although, there is still that shelf clip bug -- could you investigate that too?
(Note that a common bug we saw was that the shapes would get clipped by some sort of invisible background of the shelf, but I _think_ that's not due to the shelf itself. Something to watch out for here. Ain't CSS fun?!)
Screen Shot 2020-12-20 at 1 42 57 AM

File: sass/main.scss [hunk 6]
```diff
@@ -940,6 +938,57 @@ footer {
...
+
+	#logo {
+	    width: 120px;
+	    height: 120px;
+	}
+
+	#mlh-badge{
+	    width: 140px;
+	}
+
```
Comment (peterkos) [inline sass/main.scss:960]: nit: extra newline here between closing braces should be removed

File: sass/main.scss [hunk 7]
```diff
@@ -782,10 +782,8 @@ footer {
     }
 }
 
-
 // Media queries for mobile
 
-
```
Comment (peterkos) [inline sass/main.scss:788]: Let's keep these newlines, helps keep the mobile section separate. Unless you think it's still clear without the extra space?

**Review by peterkos — CHANGES_REQUESTED** [review]

File: sass/main.scss (hunk 4 above)
Comment (peterkos) [reply sass/main.scss:964]: Note: shelf clipping is a separate bug: #1070

File: sass/main.scss (hunk 7 above)
Comment (sjv01180) [reply sass/main.scss:788]: [should be fixed here](https://github.com/codeRIT/brickhack.io/pull/1053/commits/aeddb50e352786a0ab75c83515170efe495ad791)

File: sass/main.scss (hunk 6 above)
Comment (sjv01180) [reply sass/main.scss]: [fixed in this commit](https://github.com/codeRIT/brickhack.io/pull/1053/commits/c36383cac36d0d0da2f36bb0f49511309066d7a6)

File: sass/main.scss (hunk 5 above)
Comment (sjv01180) [reply sass/main.scss]: [resolved within this commit](https://github.com/codeRIT/brickhack.io/pull/1053/commits/cb54bb2673be9e092417dba2a09e1423c7d038aa)

File: sass/main.scss (hunk 4 above)
Comment (sjv01180) [reply sass/main.scss]: [fixed here](https://github.com/codeRIT/brickhack.io/pull/1053/commits/cb54bb2673be9e092417dba2a09e1423c7d038aa)

File: sass/main.scss (hunk 2 above)
Comment (sjv01180) [reply sass/main.scss]: [removed in this commit](https://github.com/codeRIT/brickhack.io/pull/1053/commits/cb54bb2673be9e092417dba2a09e1423c7d038aa)

File: sass/main.scss (hunk 3 above)
Comment (sjv01180) [reply sass/main.scss]: [done](https://github.com/codeRIT/brickhack.io/pull/1053/commits/c36383cac36d0d0da2f36bb0f49511309066d7a6)

File: sass/main.scss (hunk 1 above)
Comment (sjv01180) [reply sass/main.scss]: [media query movement](https://github.com/codeRIT/brickhack.io/pull/1053/commits/0c7065e5da04c118920cccd5cbda026c2aa7c512)

spacing should be fixed on the most recent commit

File: sass/main.scss [hunk 8]
```diff
@@ -940,8 +966,10 @@ footer {
         grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
+
```
Comment (peterkos) [inline sass/main.scss:969]: nit: No newline

File: sass/main.scss [hunk 9]
```diff
@@ -940,8 +966,10 @@ footer {
...
 }
 
+
```
Comment (peterkos) [inline sass/main.scss:972]: nit: No newline
(between media queries let's keep just one newline)

**Review by peterkos — CHANGES_REQUESTED** [review]

**Review by peterkos — APPROVED** [review]

**Pull request merged**